From 706f2bca7d66b2a15d8d59d8221888df663921d7 Mon Sep 17 00:00:00 2001
From: daniml3 <danimoral1001@gmail.com>
Date: Sun, 07 Mar 2021 21:03:20 +0100
Subject: [PATCH] Trebuchet: Adjust the icon size to the grid size

[ghostrider-reborn: also adjust text sizes]

- Hotseats for 4_by_5 is 4

Co-authored-by: Adithya R <gh0strider.2k18.reborn@gmail.com>
Change-Id: I02ff29cc14e5d4288a34dbd8c4658f7cea7d579f
Signed-off-by: althafvly <althafvly@gmail.com>
---

diff --git a/res/xml/device_profiles.xml b/res/xml/device_profiles.xml
index c1fa8d1..cc33cd4 100644
--- a/res/xml/device_profiles.xml
+++ b/res/xml/device_profiles.xml
@@ -21,6 +21,7 @@
         launcher:name="2_by_2"
         launcher:numRows="2"
         launcher:numColumns="2"
+        launcher:numAllAppsColumns="4"
         launcher:numFolderRows="2"
         launcher:numFolderColumns="2"
         launcher:numHotseatIcons="2"
@@ -32,16 +33,20 @@
             launcher:name="Super Short Stubby"
             launcher:minWidthDps="200"
             launcher:minHeightDps="200"
-            launcher:iconImageSize="48"
-            launcher:iconTextSize="12.0"
+            launcher:iconImageSize="120"
+            launcher:iconTextSize="17.0"
+            launcher:allAppsIconSize="60.0"
+            launcher:allAppsIconTextSize="14.0"
             launcher:canBeDefault="true" />
 
         <display-option
             launcher:name="Shorter Stubby"
             launcher:minWidthDps="200"
             launcher:minHeightDps="300"
-            launcher:iconImageSize="48"
-            launcher:iconTextSize="12.0"
+            launcher:iconImageSize="120"
+            launcher:iconTextSize="17.0"
+            launcher:allAppsIconSize="60"
+            launcher:allAppsIconTextSize="14.0"
             launcher:canBeDefault="true" />
 
     </grid-option>
@@ -50,6 +55,7 @@
         launcher:name="3_by_3"
         launcher:numRows="3"
         launcher:numColumns="3"
+        launcher:numAllAppsColumns="4"
         launcher:numFolderRows="2"
         launcher:numFolderColumns="3"
         launcher:numHotseatIcons="3"
@@ -61,16 +67,20 @@
             launcher:name="Super Short Stubby"
             launcher:minWidthDps="255"
             launcher:minHeightDps="300"
-            launcher:iconImageSize="48"
-            launcher:iconTextSize="12.0"
+            launcher:iconImageSize="85"
+            launcher:iconTextSize="15.0"
+            launcher:allAppsIconSize="60"
+            launcher:allAppsIconTextSize="14.0"
             launcher:canBeDefault="true" />
 
         <display-option
             launcher:name="Shorter Stubby"
             launcher:minWidthDps="255"
             launcher:minHeightDps="400"
-            launcher:iconImageSize="48"
-            launcher:iconTextSize="12.0"
+            launcher:iconImageSize="85"
+            launcher:iconTextSize="15.0"
+            launcher:allAppsIconSize="60"
+            launcher:allAppsIconTextSize="14.0"
             launcher:canBeDefault="true" />
 
     </grid-option>
@@ -79,9 +89,9 @@
         launcher:name="4_by_5"
         launcher:numRows="5"
         launcher:numColumns="4"
-        launcher:numFolderRows="4"
-        launcher:numFolderColumns="4"
-        launcher:numHotseatIcons="5"
+        launcher:numFolderRows="3"
+        launcher:numFolderColumns="3"
+        launcher:numHotseatIcons="4"
         launcher:dbFile="launcher_4_by_5.db"
         launcher:defaultLayoutId="@xml/default_workspace_4x5"
         launcher:deviceCategory="phone|multi_display" >
@@ -90,40 +100,40 @@
             launcher:name="Short Stubby"
             launcher:minWidthDps="275"
             launcher:minHeightDps="420"
-            launcher:iconImageSize="46"
-            launcher:iconTextSize="12.0"
+            launcher:iconImageSize="65"
+            launcher:iconTextSize="13.0"
             launcher:canBeDefault="true" />
 
         <display-option
             launcher:name="Stubby"
             launcher:minWidthDps="255"
             launcher:minHeightDps="450"
-            launcher:iconImageSize="46"
-            launcher:iconTextSize="12.0"
+            launcher:iconImageSize="65"
+            launcher:iconTextSize="13.0"
             launcher:canBeDefault="true" />
 
         <display-option
             launcher:name="Nexus S"
             launcher:minWidthDps="296"
             launcher:minHeightDps="491.33"
-            launcher:iconImageSize="46"
-            launcher:iconTextSize="12.0"
+            launcher:iconImageSize="65"
+            launcher:iconTextSize="13.0"
             launcher:canBeDefault="true" />
 
         <display-option
             launcher:name="Nexus 4"
             launcher:minWidthDps="359"
             launcher:minHeightDps="567"
-            launcher:iconImageSize="50"
-            launcher:iconTextSize="12.0"
+            launcher:iconImageSize="65"
+            launcher:iconTextSize="13.0"
             launcher:canBeDefault="true" />
 
         <display-option
             launcher:name="Nexus 5"
             launcher:minWidthDps="335"
             launcher:minHeightDps="567"
-            launcher:iconImageSize="50"
-            launcher:iconTextSize="12.0"
+            launcher:iconImageSize="65"
+            launcher:iconTextSize="13.0"
             launcher:canBeDefault="true" />
 
     </grid-option>
